ui/components/app/snaps/snap-ui-selector/snap-ui-selector.tsx

```diff
@@ -141,7 +172,11 @@ export const SnapUISelector: React.FunctionComponent<SnapUISelectorProps> = ({
       >
         {label && <Label htmlFor={name}>{label}</Label>}
         <ButtonBase
-          className="snap-ui-renderer__selector"
+          className={
+            className
+              ? classnames('snap-ui-renderer__selector', className)
```

Comment:
Doesn't classnames omit undefined this by default? I don't think we need the ternary

Reply:
Not in Typescript apparently 

```
Argument of type 'string | undefined' is not assignable to parameter of type 'string | Record<string, boolean>'.
  Type 'undefined' is not assignable to type 'string | Record<string, boolean>'.ts(2345)
```

Reply:
https://github.com/MetaMask/metamask-extension/blob/main/types/classnames.d.ts I think our type is wrong

Reply:
I'm not touching this here :joy: